### PR TITLE
Removed BrowserSync from depenencies

### PR DESCRIPTION
**Pull request by dadadavorin**
# Description

Removed BrowserSync as a project dependency.

I searched the whole project for potential usages and found none, and ran linters as well to detect possible issues.

Also updated ES lint config files based on a new setup.

### CODE REVIEW

**Review by iruzevic — CHANGES_REQUESTED** [review]
you need to change the package.json version and provide the change log details for the PR to valid

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
...
 
+## [12.1.5]
+
+### Removed
+- Removed BrowserSync dependencies.
+- Removed Babel dependencies.
+
+### Changed
+- Updated ES lint config and added required dependencies for a new setup.
+
```
Comment (iruzevic) [inline CHANGELOG.md:15]: you are missing the change tag in the bottom of this file

**Review by iruzevic — APPROVED** [review]

**Review by iruzevic — APPROVED** [review]

File: linters/ignore-gitignored.mjs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import path from 'node:path';
+import { includeIgnoreFile } from '@eslint/compat';
+import { fileURLToPath } from 'node:url';
+
+const filename = fileURLToPath(import.meta.url);
+const dirname = path.dirname(filename);
+const gitignorePath = path.resolve(dirname, '.gitignore');
+
+export default [includeIgnoreFile(gitignorePath)];
```
Comment (goranalkovic-infinum) [inline linters/ignore-gitignored.mjs:9]:
```suggestion
export default [includeIgnoreFile(gitignorePath)];

```

File: package.json [hunk 3]
```diff
@@ -77,6 +74,7 @@
 		"postcss": "^8.4.38",
 		"postcss-loader": "^8.1.1",
 		"postcss-scss": "^4.0.9",
+		"prettier": "^3.3.2",
```
Comment (goranalkovic-infinum) [inline package.json:77]: Remove all Prettier-related things, it wouldn't be too wise to add them to the existing setup (there would probably be a loooot of re-formatting 😄)

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]

**Review by piqusy — APPROVED** [review]
LGTM

**Review by goranalkovic-infinum — APPROVED** [review]

**Pull request merged**